tests/unit/type/list.tcl

```diff
@@ -599,6 +599,35 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
       $rd read
     } {}
 
+    test {SWAPDB awakes blocked client} {
+        r flushall
+        r select 1
+        r rpush k hello
+        r select 9
+        set rd [redis_deferring_client]
+        $rd brpop k 5
+        wait_for_blocked_clients_count 1
+        r swapdb 1 9
+        $rd read
+    } {k hello} {singledb:skip}
+
+    test {SWAPDB awakes blocked client, but the key already expired} {
+        r flushall
+        r debug set-active-expire 0
+        r select 1
+        r rpush k hello
+        r pexpire k 100
+        r select 9
+        set rd [redis_deferring_client]
+        $rd brpop k 1
```

Comment:
we're relying on the timeout.. i don't like it that much.
maybe instead add an assertion to confirm the client is still blocked and then CLIENT UNBLOCK it?